### DISCUSSION TITLE
delete pnpm and pnpm

### DISCUSSION THREAD

**Discussion by q306038470** [pnpm/pnpm#4991] [Q&A]
I want to uninstall pnpm, but after uninstalling, there is still a version of pnpm, and then I cannot reinstall it.

![image](https://user-images.githubusercontent.com/30361914/177706647-2c681fff-d03b-4531-b039-b0fa725dbcc7.png)

**Comment by zkochan** [comment]
Maybe you used different installation methods. Try to uninstall with npm. Or find the location with `which pnpm` and remove manually.
